### ISSUE TITLE
External CSS

### ISSUE THREAD

**Issue by lexxvlad** [mpdf/mpdf#1647]
<link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" rel="stylesheet">
    <style>
        }
        .table_pad td {
            padding: 02px;
        }
    </style>
</head>
-->

### I found this bug
mpdf 8.0.17 was correct PDF file. From version 8.1+ I can add or remove external CSS - no any chnages in result PDF file. For example text font did not changed to Sans Serif as before.

### This is mPDF and PHP version and environment (server/fpm/cli etc) I am using
mpdf 8.1.1
php 7.4.27

### This is the PHP code snippet I use

```
<?php
			$mpdf = new Mpdf(['tempDir' => storage_path('tmp')]);
			$mpdf->WriteHTML($html);
			$mpdf->Output();


```

### This is the HTML/CSS code snippet I use

```

```

**Comment by finwe** [comment]
Please, provide an actual reproducible example code. The provided code does not output anything visible. Also, I am not sure I entirely understand your description, can you be more descriptive?

**Comment by lexxvlad** [comment]
Full example code
============
`<html>
<head>
    
    <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" rel="stylesheet">
    <style>
        }
        .table_pad td {
            padding: 02px;
        }
    </style>
</head>
<body>
<div>
    <div class="container-fluid">
        {{--HEADER--}}
            <div class="row">
                <h1>Test title</h1>
            </div>
    </div>
</div>
</body>
</html>
`
==========

[mpdf 8.0.17 pdf file](https://drive.google.com/file/d/1gj-uuGgYZ6R_QAtc1R0XAYyjXWE-5q5Q/view?usp=sharing)

[mpdf 8.1.1 pdf file](https://drive.google.com/file/d/1oNwkDlUjQz0LwphAz_TZtcVXf_AgNEdZ/view?usp=sharing)

**Comment by lexxvlad** [comment]
> Files are not accessible, they are set as private

Sorry, check now pls

**Comment by oradwell** [comment]
@finwe thanks for the fix. when's this going to be released? we cannot upgrade to 8.1 until this fix is released
